tests/pyoozie/test_coordinator.py

```diff
@@ -1,16 +1,16 @@
 # Copyright (c) 2017 "Shopify inc." All rights reserved.
 # Use of this source code is governed by a MIT-style license that can be found in the LICENSE file.
-from datetime import datetime, timedelta
+import datetime
 
 import pytest
+import tests.utils
 
-from pyoozie.coordinator import Coordinator, ExecutionOrder, Configuration, Parameters
-from pyoozie.tags import MAX_NAME_LENGTH
-from tests.utils import xml_to_dict_unordered
+from pyoozie import coordinator
+from pyoozie import tags
 
 
 def parse_datetime(string):
-    return datetime.strptime(string, '%Y-%m-%dT%H:%MZ')
+    return datetime.datetime.strptime(string, '%Y-%m-%dT%H:%MZ')
```

Comment:
Of all the symbol->module import changes, this is the only one that bugs me. 
Note to self: never fall into the temptation of naming a module and class the same thing.